### DISCUSSION TITLE
Amazon Music - Prime Members

### DISCUSSION THREAD

**Discussion by BristolBEAT** [music-assistant/support#978] [Feature requests and ideas]
Given Amazon Music made this free (some what) to Prime members yesterday. Is this now an easier implement?

**Comment by sambarlick** [comment]
Definitely, this should be included.

**Comment by formatBCE** [comment]
+1 and bump. Really good addition to music collection.

**Comment by OzGav** [comment]
See original thread https://github.com/orgs/music-assistant/discussions/626
